python/tests/utils/test_geomserde_speedup.py

```diff
@@ -135,6 +137,15 @@ def test_geometry_collection(self):
         ]
         self._test_serde_roundtrip(geometry_collections)
 
+    @pytest.mark.skipif(
+        shapely.__version__ < "2", reason="SRID functions require Shapely >= 2.0"
+    )
```

Comment:
The shapely requirement of Sedona is >= 1.7.0. I found that the newly added geopandas support depends on features only in Shapely 2, this calls out for the removal of Shapely 1.x support.